faqs/galaxy/histories_purging_histories.md

```diff
@@ -0,0 +1,16 @@
+---
+title: Purging unneeded histories
+area: histories     
+box_type: tip        
+layout: faq        
+contributors: [nurzhamalyrys] 
```

Comment:
```suggestion
contributors: [jennaj, nurzhamalyrys] 
```

Reply:
@Nurzhamalyrys you can commit directly in the suggestion here, no need to edit the file